params/config.go

```diff
@@ -412,6 +412,9 @@ type NodeConfig struct {
 	// LogFile is filename where exposed logs get written to
 	LogFile string
 
+	// RuntimeLoglevel defines minimum log level for this session only, not affecting the db-stored node configuration
+	RuntimeLogLevel string `validate:"omitempty,eq=ERROR|eq=WARN|eq=INFO|eq=DEBUG|eq=TRACE"`
```

Comment:
How does it work? I don't see it being used anywhere other than in tests :thinking:

Reply:
> How does it work? I don't see it being used anywhere other than in tests 🤔

Huh, made some mistakes, fixing this.

Reply:
We set it from desktop if any runtime options are found:
https://github.com/status-im/status-desktop/pull/13309/files#diff-92c30cc52000ad1754ec5845cb656b2d8fe5a050db412429093cbdd9bb83d2e1